### ISSUE TITLE
"Cannot GET / " issue on httpAdminRoot 

### ISSUE THREAD

**Issue by No-CQRT** [node-red/node-red#842]
Just testing the **httpAdminRoot** feature on settings.js
Works fine, but now, by default, server root give a "Cannot GET / "
This error, of course appear also on "normal mode"  for a wrong address ( like 127.0.0.1:1880/foo ), but, in this case is more annoying, due default address is empty.
It will be possible to set a redirect or a error handling page?
I know that is possible to serve a static page for root, with **httpStatic** but the error will be remain for wrong requests.

**Comment by knolleary** [comment]
I don't think we should redirect as this may not always be desirable - for example if someone has moved the editor to a hidden url not intending it to be discoverable, an auto-redirect would be counter-productive. Both that and a custom 404 page _could_ be an optional behaviour - more of a debug tool than an 'in production' thing.

Needs some more thinking about to weigh up what actual benefits it would bring.

**Comment by knolleary** [comment]
Closing as part of a tidy-up of the issue tracker — this has been inactive for a long time and much has changed in Node-RED since. If it's still relevant on a current version, please open a fresh issue with up-to-date details and we'll take a look. Thanks for raising it.
